test/orchard/inspect_test.clj

```diff
@@ -941,32 +958,6 @@
                         (:newline))
                       (datafy-section rendered))))))))
 
-(deftest inspect-navigable-metadata-extension-test
```

Comment:
Test deleted as it was based on eagerly calling `nav`.

The added test already shows the behavior for when calling nav intentionally.